boards/FSM/Src/App/App_FlowMeter.c

```diff
@@ -28,3 +31,17 @@ float App_FlowMeter_ReadFlowRate(struct FlowMeter *flow_meter)
 {
     return flow_meter->get_flow_rate();
```

Comment:
I thought I left a comment about this, but we need a field in `sturct FlowMeter` for `float flow_rate`, and it would get updated every tick. 
```
float App_FloatMeter_GetFlowMeter(flow_meter) {
    return flow_meter->flow_rate
}
```

```
App_FlowMeter_Tick(flow_meter) {
    flow_meter->flow_rate = flow_meter->get_flow_rate(); // Update flow_rate each tick
}
```

Reply:
fixed